assets/js/modules/adsense/datastore/urlchannels.js

```diff
@@ -28,13 +28,11 @@ import { __ } from '@wordpress/i18n';
 import API from 'googlesitekit-api';
 import Data from 'googlesitekit-data';
 import { STORE_NAME } from './constants';
-import { parseAccountID } from '../util';
 import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
 
 const fetchGetURLChannelsStore = createFetchStore( {
 	baseName: 'getURLChannels',
-	controlCallback: ( { clientID } ) => {
-		const accountID = parseAccountID( clientID );
+	controlCallback: ( { accountID, clientID } ) => {
 		if ( undefined === accountID ) {
 			// Mirror the API response that would happen for an invalid client ID.
 			return new Promise( () => {
```

Comment:
This entire clause should now be removed. It was only relevant if `accountID` couldn't be parsed from `clientID`, which is now no longer needed.

Reply:
Ah okay, that's kind of what I thought but I wasn't sure 👍